src/components/PastTripList.js

```diff
@@ -1,20 +1,36 @@
 import styled from 'styled-components';
+import Button from './Button';
 
-export default function PastTripList({ country, city, _id }) {
+export default function PastTripList({ country, city, _id, image }) {
   return (
     <PastTripLists key={_id}>
-      {country}, {city}
+      <Image alt="country" src={image} />
+      <Place>
+        {country}, {city}
+      </Place>
+      <Button>Notes</Button>
```

Comment:
This Button is rather useless, is it? This should not be the end of a user story, since the user cannot use the product the way it looks.

Reply:
Hello Paul, thank you for the code review and helpful comments. If you click on the "Notes Button", there will be a form where the user is able to add notes to the respective cards.